cinema/models.py

```diff
@@ -1,6 +1,11 @@
+import pathlib
+import uuid
+import os
```

Comment:
Code Style Issue: You have imported `os` but it seems you are not using it anywhere in the code. It's a good practice to remove unused imports.